tests/validation/tests/v3_api/test_airgap_ha.py

```diff
@@ -226,8 +228,8 @@ def add_rancher_images_to_private_registry(
 
     # Remove the "docker save" and "docker load" lines to save time
```

Comment:
nit. This needs to be changed to `Comment` not remove if this is now commenting the line.